test_runner/src/main/kotlin/ftl/gc/GcTestMatrix.kt

```diff
@@ -48,4 +49,21 @@ object GcTestMatrix {
 
         throw RuntimeException("Failed to refresh matrix")
     }
+
+    fun cancel(testMatrixId: String, config: IArgs): CancelTestMatrixResponse {
+        val cancelMatrix = GcTesting.get.projects().testMatrices().cancel(config.projectId, testMatrixId)
```

Comment:
if we need the projectId, it may make sense to add that in the `matrix_id.json` serialization, that'll decouple the cancel logic from the configuration.